html_reference.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<link rel="stylesheet" href="css_stylesheet.css">
+		<script src="https://kit.fontawesome.com/c90803ff49.js" crossorigin="anonymous"></script>
+		<title>Milo Girardet</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+	</head>
+	<body onTouchStart="">
+	<header class="contactDetails">
+		<div class="container">
+			<div class="buttonsSentLeft">
+				<ul class="contactAnchors">
+					<li><a href="#" class="contactAnchorsStyling">milogirardet@gmail.com</a></li>
```

Comment:
You can add a link into here, if you search for 'mailto links' you should find how. That way when someone clicks on your email address it will open up their email client and start an email to you.

Reply:
Cool. Added, thank you!